students/arkadiusz_kasprzyk/lesson_07_strings/number_of_words.py

```diff
@@ -0,0 +1,11 @@
+# number_of_words.py
+
+
+def words_number(string):
+    nr = string.count(" ") + 1
```

Comment:
That may fail in many ways. If you use `len(string.split())`, then it will handle newlines, tabs, and multiple spaces at least. There may still be dashes, that would fail, but for that we would probably need a regex, not in this lesson.

Reply:
That's all true. It was written many weeks ago when I didn't know that .split() is so "intelligent".